modules/extensions-spring-support/src/test/resources/models/heisenberg.json

```diff
@@ -16115,10 +16115,9 @@
                         "parent": "",
                         "genericTypes": []
                       },
-                      "xmlHints": {
+                      "typeXmlHints": {
```

Comment:
is this rename actually necessary? is there a clash with any other type annotation?